schema/artist/index.js

```diff
@@ -88,6 +94,38 @@ const ArtistType = new GraphQLObjectType({
               type: GraphQLInt,
               resolve: ({ auction_lots_count }) => auction_lots_count,
             },
+            forsale_artworks_count: {
```

Comment:
I would rename this as `for_sale_artworks`. the `count` is implicit in the parent field